tests/framework/installer/ceph_manifests.go

```diff
@@ -221,6 +221,18 @@ spec:
                 deviceFilter: {}
                 config: {}
                 storageClassDeviceSets: {}
+            driveGroups:
```

Comment:
After we have an octopus build with drive groups, we need an integration test that uses them. Let's pick a test suite and use drive groups in that suite. For example, the helm suite should be a good fit where we don't yet have any special config for OSDs.